wled00/wled.cpp

```diff
@@ -610,6 +610,9 @@ void WLED::initInterfaces()
 {
   DEBUG_PRINTLN(F("Init STA interfaces"));
 
+  Serial.print("IP address: ");
```

Comment:
IP address is printed out as part of the debug output when connection is established.
No need to print it separately.

Reply:
How can you see this?  If I connect with `picocom -b 115200` but that gives corrupted initial output, but that is the correct baud for later Serial output.  What settings get the initial debug output?  And can they be made the same as the later serial output so you can see both?

Reply:
(I'm using an ESP32 D1 Mini.  I see suggestions the initial baud depends on the crystal clock rate or something...?)